source/_posts/2023-10-04-release-202310.markdown

```diff
@@ -0,0 +1,405 @@
+
+## A beautiful updated logo!
+
+In case you've missed it, we have refreshed the Home Assistant logo! 🤩
+
+<img class='no-shadow' src='/images/blog/2023-09-ha10/home-assistant-logo-new.png' alt='The new Home Assistant logo on a gray background.'>
+
+This was done for the [10th anniversary of Home Assistant](/blog/2023/09/17/10-years-home-assistant/), and you can [read all about this iteration in this blog post](/blog/2023/09/17/a-refreshed-logo-for-home-assistant/). You will start seeing this logo in many more places in the upcoming weeks, but, it is already included this release!
+
+<img class='no-shadow' src='/images/blog/2023-10/home-assistant-about.png' alt='Screenshot showing the new logo showing on the about page in the Home Assistant interface.'>
+
+## Brand new My Home Assistant buttons
+
+It is not just the logo that got a facelift; we have beautiful new buttons for [My Home Assistant] as well 🤩.
+
+<center>
+<a href="https://my.home-assistant.io" target="_blank"><img class='no-shadow' src='https://my.home-assistant.io/badges/homeassistant.svg' alt='My Home Assistant logo as a button'></a>
+</center><br />
+
+If you're unfamiliar with My Home Assistant, it's a free online tool that enables websites, documentation, chat platforms, and blogs to link to places on your Home Assistant server without knowing its address or being exposed to the internet. This ensures your privacy while still allowing a seamless experience.
+
+In case you don't know what My Home Assistant is: It is a free online tool, that allows websites, documentation, chat, and blogs like these, to link into places in the Home Assistant server you run at your home, without them actually knowing the address of it and without being exposed to the internet, retaining your privacy.
+
+In our documentation you'll find those buttons to, for example, add a new integration to your Home Assistant with a single click! Try it yourself! This button below will open up your Home Assistant's interface and shows the integrations
+you have installed:
+
+<center>
+
+{% my integrations badge %}
+
+</center>
+
+So, if you see these buttons showing up somewhere, you now know what they do! You can easily create your own buttons and links to share on the [create link] page.
+
+[create link]: https://my.home-assistant.io/create-link/
+[My Home Assistant]: https://my.home-assistant.io
+
+## New tile card features for climate entities
+
+In the last release, we added [lots of new features for the tile card](/blog/2023/09/06/release-20239/#lots-of-new-tile-features),
+including features specifically for controlling your climate devices. This release, we are adding even more!
+
+[@Weissnix4711] contributed a feature that allows you to add buttons for the presets of your climate device to the tile card. Next,[@piitaya] took it a step further and added the ability to select which presets you want to show and allow showing them in a dropdown list instead of a series of buttons.
```

Comment:
```suggestion
[@Weissnix4711] contributed a feature that allows you to add buttons for the presets of your climate device to the tile card. Next,[@piitaya] took it a step further: You can now select which presets you want to show and if you want them in a dropdown list instead of a series of buttons.
```